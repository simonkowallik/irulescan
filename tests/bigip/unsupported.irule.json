[{"filepath":"unsupported.irule.json","warning":[],"dangerous":[]}]
