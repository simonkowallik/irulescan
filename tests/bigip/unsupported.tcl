


auto_execok
auto_import
auto_load
auto_mkindex
auto_mkindex_old
auto_qualify
auto_reset
bgerror
cd
dict
encoding
eof
exec
exit
fblocked
fconfigure
fcopy
file
fileevent
filename
flush
gets
glob
http
interp
load
lrepeat
lreverse
memory
namespace
open
package
pid
pkg_mkIndex
pkg::create
pwd
rename
seek
socket
source
tcl_findLibrary
tell
time
trace
unknown
update