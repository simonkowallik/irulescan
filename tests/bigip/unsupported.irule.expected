WARNING: Use of unsupported command: at `auto_execok` in `auto_execok`
WARNING: Use of unsupported command: at `auto_import` in `auto_import`
WARNING: Use of unsupported command: at `auto_load` in `auto_load`
WARNING: Use of unsupported command: at `auto_mkindex` in `auto_mkindex`
WARNING: Use of unsupported command: at `auto_mkindex_old` in `auto_mkindex_old`
WARNING: Use of unsupported command: at `auto_qualify` in `auto_qualify`
WARNING: Use of unsupported command: at `auto_reset` in `auto_reset`
WARNING: Use of unsupported command: at `bgerror` in `bgerror`
WARNING: Use of unsupported command: at `cd` in `cd`
WARNING: Use of unsupported command: at `dict` in `dict`
WARNING: Use of unsupported command: at `encoding` in `encoding`
WARNING: Use of unsupported command: at `eof` in `eof`
WARNING: Use of unsupported command: at `exec` in `exec`
WARNING: Use of unsupported command: at `exit` in `exit`
WARNING: Use of unsupported command: at `fblocked` in `fblocked`
WARNING: Use of unsupported command: at `fconfigure` in `fconfigure`
WARNING: Use of unsupported command: at `fcopy` in `fcopy`
WARNING: Use of unsupported command: at `file` in `file`
WARNING: Use of unsupported command: at `fileevent` in `fileevent`
WARNING: Use of unsupported command: at `filename` in `filename`
WARNING: Use of unsupported command: at `flush` in `flush`
WARNING: Use of unsupported command: at `gets` in `gets`
WARNING: Use of unsupported command: at `glob` in `glob`
WARNING: Use of unsupported command: at `http` in `http`
WARNING: Use of unsupported command: at `interp` in `interp`
WARNING: Use of unsupported command: at `load` in `load`
WARNING: Use of unsupported command: at `lrepeat` in `lrepeat`
WARNING: Use of unsupported command: at `lreverse` in `lreverse`
WARNING: Use of unsupported command: at `memory` in `memory`
WARNING: Use of unsupported command: at `namespace` in `namespace`
WARNING: Use of unsupported command: at `open` in `open`
WARNING: Use of unsupported command: at `package` in `package`
WARNING: Use of unsupported command: at `pid` in `pid`
WARNING: Use of unsupported command: at `pkg_mkIndex` in `pkg_mkIndex`
WARNING: Use of unsupported command: at `pkg::create` in `pkg::create`
WARNING: Use of unsupported command: at `pwd` in `pwd`
WARNING: Use of unsupported command: at `rename` in `rename`
WARNING: Use of unsupported command: at `seek` in `seek`
WARNING: Use of unsupported command: at `socket` in `socket`
WARNING: Use of unsupported command: at `source` in `source`
WARNING: Use of unsupported command: at `tcl_findLibrary` in `tcl_findLibrary`
WARNING: Use of unsupported command: at `tell` in `tell`
WARNING: Use of unsupported command: at `time` in `time`
WARNING: Use of unsupported command: at `trace` in `trace`
WARNING: Use of unsupported command: at `unknown` in `unknown`
WARNING: Use of unsupported command: at `update` in `update`

